pages/common/sdk.md

```diff
@@ -0,0 +1,28 @@
+# sdk
+
+> Tool for managing parallel versions of multiple Software Development Kits.
+> Supports Java, Groovy, Scala, Kotlin, Gradle, Maven, Vert.x and many others.
+
+- Install a specific version of Gradle:
+
+`sdk install gradle {{gradle_version}}`
+
+- Switch to a specific version of Gradle:
+
+`sdk use gradle {{gradle_version}}`
+
+- Check current Gradle version:
+
+`sdk current gradle`
+
+- List all available Software Development Kits:
+
+`sdk list`
```

Comment:
Does this list installed SDKs or ones that are available to install? Or both?

Reply:
@pxgamer Removed ambiguity